conda.recipe/meta.yaml

```diff
@@ -31,6 +31,7 @@ requirements:
 
 test:
   requires:
+    - avro
```

Comment:
Since this package isn't python 3 compatible, specify it with a [conda selector](http://conda.pydata.org/docs/building/meta-yaml.html#preprocessing-selectors) like this:

```yaml
- avro  # [py27]
```